index.js

```diff
@@ -45,19 +47,26 @@ module.exports = function (content) {
         .replace(new RegExp(escapeRegExp(pathSep) + '+', 'g'), pathSep);
     var html;
 
-    if (content.match(/^module\.exports/)) {
+    if (content.match(/(?:^module\.exports)|(?:^export\s+default)|(?:^exports.default)/)) {
```

Comment:
`/^(module\.exports|export\s+default|exports.default)/` is a bit easier to read.